src/backend/web/static/swagger/api_v3.json

```diff
@@ -5549,8 +5566,36 @@
             "format": "int64"
           },
           "score_breakdown": {
-            "type": "object",
-            "properties": {},
+            "oneOf": [
+              {
+                "$ref": "#/components/schemas/Match_Score_Breakdown_2015_Alliance"
+              },
+              {
+                "$ref": "#/components/schemas/Match_Score_Breakdown_2016_Alliance"
+              },
+              {
+                "$ref": "#/components/schemas/Match_Score_Breakdown_2017_Alliance"
+              },
+              {
+                "$ref": "#/components/schemas/Match_Score_Breakdown_2018_Alliance"
+              },
+              {
+                "$ref": "#/components/schemas/Match_Score_Breakdown_2019_Alliance"
+              },
+              {
+                "$ref": "#/components/schemas/Match_Score_Breakdown_2020_Alliance"
+              },
+              {
+                "$ref": "#/components/schemas/Match_Score_Breakdown_2022_Alliance"
+              },
+              {
+                "$ref": "#/components/schemas/Match_Score_Breakdown_2023_Alliance"
+              },
+              {
+                "$ref": "#/components/schemas/Match_Score_Breakdown_2024_Alliance"
```

Comment:
Should these be `"#/components/schemas/Match_Score_Breakdown_2024"` rather than `"#/components/schemas/Match_Score_Breakdown_2024_Alliance"`?